dex/networks/eth/tokens.go

```diff
@@ -161,8 +161,21 @@ var Tokens = map[uint32]*Token{
 		},
 		NetTokens: map[dex.Network]*NetToken{
 			dex.Mainnet: {
-				Address:       common.Address{},
-				SwapContracts: map[uint32]*SwapContract{},
+				Address: common.HexToAddress("0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48"), // https://etherscan.io/address/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48
+				SwapContracts: map[uint32]*SwapContract{
+					0: { // https://etherscan.io/address/0xa7af47db3296206ea543a82ffbf7fc312698e6c9
+						Address: common.HexToAddress("0xA7Af47DB3296206eA543A82ffBF7Fc312698E6C9"),
+						Gas: Gases{ // Same as Goerli gases. DRAFT PR: verify!
```

Comment:
This is where `GetGasEstimates` gets real. What do you suggest we use for `maxSwaps`. cc @JoeGruffins

Reply:
Do you mean mean `maxSwapsInTx`?

Reply:
That one gets computed automatically on wallet construction, based on the hard coded gases, right?

Reply:
Yes.

Reply:
Not sure what `maxSwaps` is.

Reply:
`maxSwaps int` is an argument to `GetGasEstimates` that specifies how many initiations and redemptions will be sent.